docs/operator-guide/index.md

```diff
@@ -1,14 +1,21 @@
-# Welcome to SciCat Operator's Manual
+# Welcome to SciCat Operator's Guide
 
 ## Overview
 
-Getting SciCat up and running at your site should be rather straight forward for a test deployment. However turning into a production ready system may involve a bit more work, because different existing systems will need to be interfaced to SciCat.
+Getting SciCat up and running at your site should be rather straight forward for a test deployment. However turning into a production ready system may involve some more work, because different existing systems will need to be interfaced to SciCat.
```

Comment:
Suggestion:

SciCat is a flexible metadata catalogue designed to be easily interfaced in to most existing infrastructure. The following guide will introduce the configuration required to integrate SciCat into common technologies.